tasks/release/linux.js

```diff
@@ -104,6 +104,42 @@ var packToDebFile = function () {
     return deferred.promise;
 };
 
+var packToRpmFile = function () {
+    var deferred = Q.defer();
+
+    var rpmFileName = packName + '.rpm';
+    var rpmPath = releasesDir.path(rpmFileName);
+
+    gulpUtil.log('Creating RPM package... (' + rpmFileName + ')');
+
+    // Preparing RPM Spec file
+    var spec = projectDir.read('resources/linux/RHEL/app.spec');
```

Comment:
In `fs-jetpack` it's reversed, so it would be `projectDir.readAsync()` were it not synchronous per: https://github.com/szwacz/fs-jetpack#promises-instead-of-callbacks